backend/src/typeorm/getDBVersion.ts

```diff
@@ -0,0 +1,15 @@
+import { getConnection } from 'typeorm'
+import { Migration } from './entity/Migration'
+
+const getDBVersion = async (): Promise<string | null> => {
+  const connection = getConnection()
+  const migrations = connection.getRepository(Migration)
+  try {
+    const dbVersion = await migrations.findOne({ order: { version: 'DESC' } })
+    return dbVersion ? dbVersion.fileName : null
+  } catch (error) {
+    return null
+  }
```

Comment:
@einhornimmond @Mogge Here you can see how the DataMapper Patters is accessing Data. The Queries are defined externally and not in the Model. For this case another way would be counter intuitive for me anyways, since this is a special case of "getLastColumnOrderedByDBVersionDesc" - doesn't sound like i'd like to have this function on the model to me.

Reply:
As far as I understand you don't use the DataMapper Pattern. 
You use the repository directly in your function. If this case is used only at one place, nothing is to say against it.